lib/tasks/refresh_region_names.rb

```diff
@@ -0,0 +1,56 @@
+class RefreshRegionNames
+  def self.call
+    abort "Cannot run in production!" if SimpleServer.env.production?
```

Comment:
I believe demo still runs as `RAILS_ENV = production`. You may want to check `SIMPLE_SERVER_ENV` instead.

There's this too:
`app/helpers/simple_server_env_helper.rb`

Reply:
Yup, that's why I'm checking `SimpleServer.env` instead of `Rails.env` 👍